.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+If the below instructions are not followed, we may be forced to close your issue.
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the latest release to make sure your issues has not already been fixed: 
+https://github.com/Azure/azure-iot-sdk-c/releases/latest 
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoT Hub name, Device Provisioning Service scope ID), etc...). Please submit a ticket and get assistance from the Microsoft support team using the following link: 
```

Comment:
>Please submit a ticket and get assistance from the Microsoft support team using the following link:  [](start = 160, length = 100)

"Please submit a ticket and get assistance from the Microsoft support team using the following link: "

Literally following this I'd always open the link :).  Could we change this to "If you need to share any of this information, please submit ..."